http.d.ts

```diff
@@ -111,14 +111,14 @@ export interface IHttpLink {
 export interface IHttpRequestBody {
   description?: string;
   required?: boolean;
-  content: IHttpContent[];
+  contents: IHttpContent[];
```

Comment:
Not sure whether word "contents" means the same as `content[]`. Some native will need to comment on this ;)

Reply:
This makes sense to me!